cli-commands/lb/post/create.rb

```diff
@@ -11,6 +11,7 @@
     on("-e", "--check-endpoint=path", "set the health check endpoint (default: #{Prog::Vnet::LoadBalancerNexus::DEFAULT_HEALTH_CHECK_ENDPOINT})")
     on("-p", "--check-protocol=proto", health_check_protocols, "set the health check protocol")
     on("-s", "--stack=stack", stacks, "set the stack")
+    on("-c", "--cert-enabled=cert_enabled", "set the certificate enabled")
```

Comment:
I think it would be better to have two options `-c` and `-C`, one to force enabling, one to force disabling. Having a option for a boolean that takes an argument is clunky, and it's not obvious what the argument format should be. In this case it would be whatever `typecast_params.bool` accepts, but there are other places where we do boolean checking where only `true` is treated as truthy.